eventfully/main.py

```diff
@@ -58,6 +59,53 @@ def index():
     return render_template('index.html')
 
 
+# Check the Cookie or redirect to log in
+@app.route("/checkAccount")
+def checkAccount():
+    userID = request.cookies.get('userID')
+    if userID:
+        return db.get_User_Data(request.cookies.get("userID"))
+    else:
+        return redirect("/login", 302)
+
+
+@app.route("/setAccount/<string:userID>/")
+def set_Cookie(userID):
+    resp = make_response(redirect("/checkAccount", 302))
+    resp.set_cookie('userID', userID)
+    return resp
+
+
+# Adding the User Data to the Database
+@app.route("/signin/add")
+def registerUser():
```

Comment:
Traceback (most recent call last):
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\peewee.py", line 3252, in execute_sql
    cursor.execute(sql, params or ())
sqlite3.IntegrityError: NOT NULL constraint failed: accountdata.password

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\flask\app.py", line 1478, in __call__
    return self.wsgi_app(environ, start_response)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\flask\app.py", line 1458, in wsgi_app
    response = self.handle_exception(e)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\flask\app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\flask\app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\flask\app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\flask\app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "h:\Programming\GitHub\Eventfully\eventfully\main.py", line 75, in registerUser
    db.add_Account(request.args.get("username"), request.args.get("password"), userID)
  File "H:\Programming\GitHub\Eventfully\eventfully\database.py", line 87, in add_Account
    AccountData.create(
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\peewee.py", line 6620, in create
    inst.save(force_insert=True)
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\peewee.py", line 6830, in save
    pk = self.insert(**field_dict).execute()
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\peewee.py", line 1972, in inner
    return method(self, database, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\peewee.py", line 2043, in execute
    return self._execute(database)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\peewee.py", line 2848, in _execute
    return super(Insert, self)._execute(database)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\peewee.py", line 2561, in _execute
    cursor = database.execute(self)
             ^^^^^^^^^^^^^^^^^^^^^^
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\peewee.py", line 3260, in execute
    return self.execute_sql(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\peewee.py", line 3250, in execute_sql
    with __exception_wrapper__:
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\peewee.py", line 3020, in __exit__
    reraise(new_type, new_type(exc_value, *exc_args), traceback)
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\peewee.py", line 192, in reraise
    raise value.with_traceback(tb)
  File "H:\Programming\GitHub\Eventfully\.venv\Lib\site-packages\peewee.py", line 3252, in execute_sql
    cursor.execute(sql, params or ())
peewee.IntegrityError: NOT NULL constraint failed: accountdata.password